task/06-conditions-n-loops-tasks.js

```diff
@@ -82,7 +103,7 @@ function getSumBetweenNumbers(n1, n2) {
  *   10,10,10 =>  true
  */
 function isTriangle(a,b,c) {
-    throw new Error('Not implemented');
+    return ((a < b+c) && (b < a+c) && (c < a+b)) ? true : false
```

Comment:
Please, spaces between operators
Please, try to follow these rules https://javascript.info/coding-style

we can just `return ((a < b + c) && (b < a + c)  &&  (c < a + b))`
Please, see https://javascript.info/logical-operators